languages/es-ES.js

```diff
@@ -1,169 +1,169 @@
+			SETTING_GATEWAY_KEY_NOEXT: (key) => `La clave '${key}' no existe en el esquema de datos.`,
+			SETTING_GATEWAY_INVALID_TYPE: 'El parámetro \'type\' debe ser o \'add\' o \'remove\'.',
+			RESOLVER_INVALID_CUSTOM: (name, type) => `${name} debe ser un nombre válido de ${type}`,
+			RESOLVER_INVALID_PIECE: (name, piece) => `${name} debe ser un nombre válido de ${piece}.`,
+			RESOLVER_INVALID_MESSAGE: (name) => `${name} debe ser una ID de mensaje válida.`,
+			RESOLVER_INVALID_USER: (name) => `${name} debe ser una mención o una ID de usuario válida.`,
+			RESOLVER_INVALID_MEMBER: (name) => `${name} debe ser una mención o una ID de usuario válida.`,
+			RESOLVER_INVALID_CHANNEL: (name) => `${name} debe ser una mención o una ID de canal válida.`,
+			RESOLVER_INVALID_EMOJI: (name) => `${name} debe ser un emoji válido.`,
+			RESOLVER_INVALID_GUILD: (name) => `${name} debe ser una ID válida de servidor.`,
+			RESOLVER_INVALID_ROLE: (name) => `${name} debe ser una mención o una ID de rol válida.`,
+			RESOLVER_INVALID_LITERAL: (name) => `Su opción no coincide con la siguiente posibilidad: ${name}`,
+			RESOLVER_INVALID_BOOL: (name) => `${name} debe ser 'true' o 'false'.`,
+			RESOLVER_INVALID_INT: (name) => `${name} debe ser un número entero.`,
+			RESOLVER_INVALID_FLOAT: (name) => `${name} debe ser un número.`,
+			RESOLVER_INVALID_REGEX_MATCH: (name, pattern) => `${name} debe seguir el patrón de la expresión regular \`${pattern}\``,
+			RESOLVER_INVALID_URL: (name) => `${name} debe ser un enlace URL válido.`,
+			RESOLVER_INVALID_DATE: (name) => `${name} debe ser una fecha válida.`,
+			RESOLVER_INVALID_DURATION: (name) => `${name} debe ser una duración válida.`,
+			RESOLVER_INVALID_TIME: (name) => `${name} debe ser una fecha o duración válida.`,
+			RESOLVER_STRING_SUFFIX: ' caracteres',
+			RESOLVER_MINMAX_EXACTLY: (name, min, suffix) => `${name} debe ser exactamente ${min}${suffix}.`,
+			RESOLVER_MINMAX_BOTH: (name, min, max, suffix) => `${name} debe estar entre ${min} y ${max}${suffix}.`,
+			RESOLVER_MINMAX_MIN: (name, min, suffix) => `${name} debe ser mayor que ${min}${suffix}.`,
+			RESOLVER_MINMAX_MAX: (name, max, suffix) => `${name} debe ser menor que ${max}${suffix}.`,
+			REACTIONHANDLER_PROMPT: '¿A qué página te gustaría saltar?',
+			COMMANDMESSAGE_MISSING: 'Faltan uno o más argumentos al final de la entrada.',
+			COMMANDMESSAGE_MISSING_REQUIRED: (name) => `${name} es un argumento requerido.`,
+			COMMANDMESSAGE_MISSING_OPTIONALS: (possibles) => `Falta una opción requerida: (${possibles})`,
+			COMMANDMESSAGE_NOMATCH: (possibles) => `Su opción no coincide con ninguna de las posibilidades: (${possibles})`,
+			MONITOR_COMMAND_HANDLER_REPROMPT: (tag, error, time) => `${tag} | **${error}** | Tienes **${time}** segundos para responder a este mensage emergente con un argumento válido. Escribe **"ABORT"** para abortar el proceso.`, // eslint-disable-line max-len

```

Comment:
This one had "Usted tiene" for formality, "Tienes" sounds very direct, but I'm open to reword this.

Reply:
But, is it supposed to sound formal? I wasn't aware, sorry.

Reply:
```suggestion
			MONITOR_COMMAND_HANDLER_REPROMPT: (tag, error, time) => `${tag} | **${error}** | Tienes **${time}** segundos para responder a este mensaje emergente con un argumento válido. Escribe **"ABORT"** para abortar el proceso.`, // eslint-disable-line max-len
```
I think I saw another "mensage" somewhere, probably near the same line.

Reply:
There are 6 `mensage`'s in total.

Reply:
> But, is it supposed to sound formal? I wasn't aware, sorry.

It's a matter of preference, doesn't really matter.



> There are 6 `mensage`'s in total.

Should be changed to [`mensaje`](https://dle.rae.es/srv/fetch?id=OvO0obY).